### PR TITLE
Newbie

### PR DESCRIPTION
**Pull request by nitinjaiswal7537**
# Related Issue
Fixes: Added in hover option to the mail

# Prosposed Changes
- Added in hover option  to the email
- changed the email and the phone icon 
Hey @garg3133  this is the new PR please review it.

### CODE REVIEW

File: templates/home/index.html [hunk 1]
```diff
@@ -3,15 +3,27 @@
 {% load static %}
 
 {% block title %} Jagrati | Home {% endblock %}
-
+JagratiWebApp\static\home\css
```
Comment (garg3133) [inline templates/home/index.html:6]: Why this path is present here?

File: templates/home/index.html [hunk 2]
```diff
@@ -3,15 +3,27 @@
...
+
+    <link rel="stylesheet" type="text/css" href='{% static "home/css/#style.css" %'>
+    
+    
+    <link href="https://unpkg.com/aos@2.3.1/dist/aos.css" rel="stylesheet">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css" integrity="sha384-oS3vJWv+0UjzBfQzYUhtDYW+Pj2yciDJxpsK1OYPAYjqT085Qq/1cq5FLXAZQ7Ay" crossorigin="anonymous">
+    <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/@glidejs/glide/dist/css/glide.core.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.4.0/css/font-awesome.min.css">
+    <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/@glidejs/glide/dist/css/glide.theme.>
+
```
Comment (garg3133) [inline templates/home/index.html:17]: All these links are duplicate.

File: templates/home/index.html [hunk 3]
```diff
@@ -3,15 +3,27 @@
...
+    <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/@glidejs/glide/dist/css/glide.theme.>
+                                                 
 <link rel="stylesheet" type="text/css" href='{% static "home/css/style.css" %}'>
 <link href="https://unpkg.com/aos@2.3.1/dist/aos.css" rel="stylesheet">
 <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css"
     integrity="sha384-oS3vJWv+0UjzBfQzYUhtDYW+Pj2yciDJxpsK1OYPAYjqT085Qq/1cq5FLXAZQ7Ay" crossorigin="anonymous">
 <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/@glidejs/glide/dist/css/glide.core.min.css">
 <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.4.0/css/font-awesome.min.css">
-<link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/@glidejs/glide/dist/css/glide.theme.min.css">
+<link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/@glidejs/glide/dist/css/glide.theme.min.css>
```
Comment (garg3133) [inline templates/home/index.html:24]: This link is duplicate.

File: templates/home/index.html [hunk 4]
```diff
@@ -364,6 +376,31 @@ <h1 style="margin: 0 0 10px; color: black;" data-aos="fade-up">Jagrati Orientati
...
+                    <p>XXXXxxxxxxx</p>
+                </div>
+                <div class="don-box3">
+                    <h3>Contact Details</h3>
+                    <p><i class="fa fa-phone-square fa-2x"></i>  +91-xxxxx-xxxxx</p>
+                    <p ><a  class="email"  href="mailto: jagrati@iiitdmj.ac.in" > <i class="fa fa-envelope-o fa-2x" ></i>jagrati@iiitdmj.ac.in</a></p>
+                    <p><i class="fas fa-map-marker-alt fa-2x"></i>  PDPM IIITDM JABALPUR</p>
+                </div>
+            </div>
+
```
Comment (garg3133) [inline templates/home/index.html:403]: This complete block of code is duplicate.

**Review by garg3133 — COMMENTED** [review]
@nitinjaiswal7537 There are still many changes that do not belong to this PR. See some of them below.

You need to remove any additional changes from this PR.

**Comment by garg3133** [comment]
Apart from all the code mentioned above, there is a whole lot of code that is duplicate.

**Comment by nitinjaiswal7537** [comment]
@garg3133 I had clone if from the present repository and i had no idea about how this also fake links and code comes from.

**Comment by garg3133** [comment]
@nitinjaiswal7537 As your actual changes are quite small, you can do the following:
* Save your actual changes to some other file.
* `git reset --hard HEAD~150` (150 - random big number)
* `git pull upstream master`

Before running the reset command, make sure you do not have any un-committed change.

**Comment by nitinjaiswal7537** [comment]
@garg3133 you can guide me how to identity the process to deal with this fake codes

**Comment by nitinjaiswal7537** [comment]
@garg3133 ok i will try to do so.

**Comment by nitinjaiswal7537** [comment]
Hey @garg3133  the steps you mentioned didn't reflect any changes .
so can you guide me some other steps in order to remove those fake links

**Comment by garg3133** [comment]
@nitinjaiswal7537 I don't see you making any force push here! The steps I told you should work (by your actual changes, I mean the changes you intend to make in this PR and not the changes the are currently present in this PR).

**Comment by nitinjaiswal7537** [comment]
@garg3133  error showing while implementing the 3rd line of code -
"git pull upstream master"

fatal: 'upstream' does not appear to be a git repository
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
So, what should i do??

**Comment by garg3133** [comment]
@nitinjaiswal7537 You don't seem to have linked the main repository with your local project. Run this command: `git remote add upstream https://github.com/garg3133/JagratiWebApp` before the third step.

**Comment by nitinjaiswal7537** [comment]
Hey @garg3133  while forcing to push the file it's showing such error:
error: failed to push some refs to 'https://github.com/nitinjaiswal7537/JagratiWebApp.git'
So ,how should i get it fixed.

**Comment by garg3133** [comment]
@nitinjaiswal7537 Can you share the screenshot?

**Comment by nitinjaiswal7537** [comment]
IMG20210413164324

**Comment by garg3133** [comment]
I think you are running this command from your master branch and you do not have any branch with the name `newbie` locally. If this is not the case, then it is really hard to debug what is going wrong.

Maybe you can delete your fork and create a new fork and push your changes to that fork?

**Pull request closed without merging**